packages/dappmanager/src/calls/wifi.ts

```diff
@@ -0,0 +1,92 @@
+import { ComposeFileEditor } from "../modules/compose/editor";
+import { logContainer } from "../modules/docker/api";
+import { listContainer } from "../modules/docker/list";
+import { CurrentWifiCredentials, WifiReport } from "../types";
+import params from "../params";
+
+const WIFI_KEY_SSID = "SSID";
+const WIFI_KEY_PASSWORD = "WPA_PASSPHRASE";
+
+/**
+ * Return wifi report
+ */
+export async function wifiReportGet(): Promise<WifiReport> {
+  const wifiContainer = await listContainer({
+    containerName: params.wifiContainerName
+  });
+  if (!wifiContainer)
+    throw Error(
+      `Wifi is not present. Container name: ${params.wifiContainerName}`
+    );
+
+  // exitCode 0 means that wifi package was manually stopped. No errors
+  if (wifiContainer.exitCode === 0) wifiContainer.state === "paused";
+
+  let report;
+  let info = "";
+
+  switch (wifiContainer.state) {
+    case "created":
+      info = "Wifi has not been initialized yet";
+    case "running":
+      info = "Wifi is currently running";
+    case "restarting":
+      info = "Wifi restarting. Wait until it starts";
+    case "dead":
+      info =
+        "Wifi service dead, you must manually remove it and install it again";
```

Comment:
Is it even possible for users to do that? Can they remove the WIFI container and install it again? How would they do that

Reply:
dead status would be a really weird and critical situation that must never happen, I have only seen that situation when trying to remove containers and these containers were partially removed.

Anyway if that happened for any reason, would be useful info for the user to report and get back the wifi